corehq/apps/export/dbaccessors.py

```diff
@@ -0,0 +1,32 @@
+
+
+def get_latest_case_export_schema_id(domain, case_type):
+    from .models import ExportDataSchema
+
+    key = [domain, 'CaseExportDataSchema', case_type]
+    result = ExportDataSchema.get_db().view(
+        'schemas_by_xmlns_or_case_type/view',
+        startkey=key + [{}],
+        endkey=key,
+        include_docs=False,
+        limit=1,
+        reduce=False,
+        descending=True,
```

Comment:
does this need to be descending? Also I would have thought they key would be unique so can't you just say `key=key`?